packages/odyssey-react/src/components/TextInput/TextInput.module.scss

```diff
@@ -81,16 +80,26 @@
   }
 }
 
-.outer {
-  position: relative;
+.prefix {
+  color: var(--PrefixTextColor);
+  padding-inline-end: var(--PaddingInline);
+}
+
+.suffix {
+  color: var(--SuffixTextColor);
+  padding-inline-start: var(--PaddingInline);
 }
```

Comment:
consider a shared declaration for the shared styles:

```
.prefix, .suffix {
  padding-inline-start: var(--PaddingInline);
  cursor: default
}
```

I added that cursor style because I noticed it changes to the selection cursor on hover which we don't want I imagine cc @edburyenegren-okta @josesolano-okta

Reply:
I'm split on cursor style. USWDS set the cursor to `default` and adds `user-select: none`. It removes some functionality for sighted users, but is better aligned with our `aria-hidden` behavior for assistive devices. I'm +1 here -- if users find it odd, we can remove the properties.

Their paddings are different, but we may want to collapse the Prefix and Suffix color variables into a shared declaration as well. @josesolano-okta would we want to theme their colors separately?

Reply:
so? use `cursor: default; `?